src/macros.php

```diff
@@ -52,6 +52,49 @@
     });
 }
 
+if (! CrudColumn::hasMacro('linkTo')) {
+    CrudColumn::macro('linkTo', function (string|array $routeNameOrConfiguration, ?string $target = null): static {
```

Comment:
Hmm why do we have `$routeNameOrConfiguration` here? From what I can tell the only configuration possible is the target, and that is already treated by a second parameter... right? Doesn't seem like we need this complication of having both.

```suggestion
    CrudColumn::macro('linkTo', function (string $routeName, ?string $target = null): static {
```

Reply:
Is it because using the array syntax, only the first parameter gets used? Can't we use the spread operator or something like that there?

Reply:
You are right, it's because of the array syntax. 

I think moving it to a spread operator would turn this PR into a BC https://github.com/Laravel-Backpack/CRUD/blob/4fca6897d29f7a91e703e5b6d8887a7d9ad900e3/src/app/Library/CrudPanel/Traits/Support/MacroableWithAttributes.php#L40C17-L40C17

We could refactor that function, at the moment it does not accept any macro value that's not an array.
It may require small changes on Uploaders constructors since they rely on receiving an array.

Reply:
Pfff... nah, fuck it, let's keep it as you have it here. If it works, it's good.